presto-docs/src/main/sphinx/release/release-0.281.rst randomize_outer_join_null_key_strategy randomize_outer_join_null_key_strategy

```diff
@@ -0,0 +1,59 @@
+=============
+Release 0.281
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix timeout for optimization rules by adding interrupt check between each optimizer rule, and throw meaningful exception.
+* Improve memory accounting for rehashing for values by keeping both the new table and the old table, before this chage we account for the only size difference between the new table and the old table.
```

Comment:
Too technical.  What is the end impact to the user?  Rehashing of tables in hash join?

Reply:
I don't think this needs a release note. But we can simplify this to - 
"Improve memory accounting for rehashing of tables in hash join"

Reply:
One other suggestion, if this is a memory accounting fix, then that's more like a bug fix.  `Fix memory accounting for hash join.`